src/mscorlib/src/System/Reflection/EventInfo.cs

```diff
@@ -396,7 +396,7 @@ public override MethodInfo[] GetOtherMethods(bool nonPublic)
             List<MethodInfo> ret = new List<MethodInfo>();
 
             if ((object)m_otherMethod == null)
-                return new MethodInfo[0];
+                return Array.Empty<MethodInfo>();
```

Comment:
GetOtherMethods doesn't exist in .NET Core, this array may be returned publically, etc.  For the same reasoning as explained elsewhere, let's revert this.